drake/systems/plants/@RigidBodyManipulator/findKinematicPath.m

```diff
@@ -20,7 +20,10 @@
 end
 
 if use_mex
-  [body_path, joint_path, signs] = findKinematicPathmex(obj.mex_model_ptr, start_body, end_body);
+  [body_path, joint_path, signs] = findKinematicPathmex(obj.mex_model_ptr, start_body - 1, end_body - 1); % base 1 to base 0
```

Comment:
That can't be maintained in general in this kind of framework. One C++ argument type needs to map to one conversion method from Matlab, at least within one function. Right now we have e.g.
```
  template <typename Scalar>
  GradientVar<Scalar, TWIST_SIZE, 1> geometricJacobianDotTimesV(const KinematicsCache<Scalar>& cache, int base_body_or_frame_ind, int end_effector_body_or_frame_ind, int expressed_in_body_or_frame_ind, int gradient_order) const;
```
where `base_body_or_frame_ind`, `end_effector_body_or_frame_ind`, `expressed_in_body_or_frame_ind`, and `gradient_order` are all ints, but `gradient_order` should not have a base 1 to base 0 conversion applied, while the others do.

I think the long term solution is to actually have these arguments be of different types, i.e. turn the `*_body_or_frame_ind` arguments into `const RigidBodyFrame&`. In fact, in this case `gradient_order` is going to disappear when we switch over to AutoDiffScalar gradients.